src/Config/FileSystemRuntimeConfigLoader.cs

```diff
@@ -56,11 +56,20 @@ public class FileSystemRuntimeConfigLoader : RuntimeConfigLoader
     public FileSystemRuntimeConfigLoader(IFileSystem fileSystem, string baseConfigFilePath = DEFAULT_CONFIG_FILE_NAME, string? connectionString = null)
         : base(connectionString)
     {
-        FileSystem = fileSystem;
+        _fileSystem = fileSystem;
         _baseConfigFilePath = baseConfigFilePath;
         ConfigFilePath = GetFinalConfigFilePath();
     }
 
+    /// <summary>
+    /// Gets the current directory of the file system and returns
+    /// it as a string.
+    /// </summary>
+    /// <returns></returns>
+    public string GetCurrentDirectoryFromFileSystem()
```

Comment:
`FromFileSystem` is an implementation detail. The caller of this method just wants to get a directory value that represents.. ? 

However, simplifying to `GetCurrentDirectory()` is vague and seems like we're exposing filesystem internals. 

Can you think of a name that would represents:
-  What the method is returning (directory)
- The method'significance/relevance to the caller? 
- What's the relation of 'current directory' to `FileSystemRuntimeConfigLoader`? Current directory of what?

Reply:
followup up comment for clarity, after our sync on teams decided to keep GetCurrentDiretoryFromFileSystem